Lib/_pylong.py

```diff
@@ -210,6 +210,121 @@ def inner(a, b):
     w5pow = compute_powers(len(s), 5, DIGLIM)
     return inner(0, len(s))
 
+# Asymptotically faster version, using the C decimal module. See
+# comments at the end of the file. This uses decimal arithmetic to
+# convert from base 10 to base 256. The latter is just a string of
+# bytes, which CPython can convert very efficiently to a Python int.
+
+# log of 10 to base 256 with best-possible 53-bit precision. Obtained
+# via:
+#    from mpmath import mp
+#    mp.prec = 1000
+#    print(float(mp.log(10, 256)).hex())
+_LOG_10_BASE_256 = float.fromhex('0x1.a934f0979a371p-2') # about 0.415
+
+def _dec_str_to_int_inner(s):
+    BYTELIM = 512
+    D = decimal.Decimal
+    result = bytearray()
+
+    def inner(n, w):
+        #assert n < D256 ** w # required, but too expensive to check
+        if w <= BYTELIM:
+            # XXX Stefan Pochmann discovered that, for 1024-bit ints,
```

Comment:
What does XXX imply here? Based on the successive sentence, I judge it is a kind of a TODO comment, but it is hard to tell. Mind spelling it out instead of adding a cryptic "XXX"?

Reply:
It's been used in CPython's source for decades.

https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments

A quick search on the `main` branch now found 976 instances in our `.py` files, 574 in `.c` files, and 131 in `.h` files. Far more common than "TODO". When in Rome .,,

Note that a PEP to impose rules on when to use which flavors of tag was rejected:

https://peps.python.org/pep-0350/

I'm going to close this too, because you're going to trip over a thousand other instances too - get used to it :smile:.